sw/device/lib/runtime/README.md

```diff
@@ -0,0 +1,5 @@
+# `libruntime`, the OpenTitan Runtime Library
+
+This directory contains runtime-aware base libraries for OpenTitan, being more
```

Comment:
Aren't they more machine-aware, or environment-aware, than runtime-aware?

Reply:
"machine aware" sounds better. I think that, at this level of execution, I tend to conflate the RISC-V core and the notion of "runtime".